axelar-chains-config/info/mainnet.json

```diff
@@ -124,7 +124,10 @@
         "gasPrice": 150000000000
       },
       "finality": "finalized",
-      "approxFinalityWaitTime": 40
+      "approxFinalityWaitTime": 40,
+      "gasOptions": {
+        "blobBaseFee": 1
```

Comment:
gasOptions are fed directly into evm txs, so unexpected values can cause it to fail. create a different field instead `onchainGasEstimate`. otherwise you'd need to delete it in the utils:
https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/evm/utils.js#L1052